src/lib.rs

```diff
@@ -808,6 +808,58 @@ impl Default for Box<CStr> {
     }
 }
 
+#[cfg(feature = "std")]
+use std::ffi::{CStr as StdCStr, CString as StdCString};
+
+#[cfg(feature = "std")]
+impl From<CString> for StdCString {
+    fn from(s: CString) -> StdCString {
+        unsafe { StdCString::from_raw(s.into_raw()) }
```

Comment:
This performs an unnecessary allocation. Prefer `into_bytes_with_nul` and `from_vec_unchecked`.